README.md

```diff
@@ -1,9 +1,17 @@
-# testing
+![X-Road logo](https://raw.githubusercontent.com/ria-ee/X-Road/develop/xroad_logo_small.png) ![European Regional Development Fund](https://github.com/e-gov/RIHA-Frontend/raw/master/logo/EU/EU.png "European Regional Development Fund - DO NOT REMOVE THIS IMAGE BEFORE 01.11.2022")
 
-This is a library for X-Road-tests for X-Road 6.16.0

```

Comment:
Do we need to keep the version information here or is it OK to delete that?

Reply:
According to [test workflow](https://github.com/vrk-kpa/xroad-joint-development/blob/master/TEST_WORKFLOW.md) the version number of X-Road the tests were designated for should be stated here, so it is not ok to remove it.